adaptive-education-ai/src/ibm-watson/IbmWatsonNluService.ts

```diff
@@ -0,0 +1,110 @@
+'use strict';
```

Comment:
We can use a typescript compiler flag to automatically put 'use strict' on every transpiled file, so we don't need to do it manually. We should do that.